crates/wasi-nn/tests/all.rs

```diff
@@ -45,13 +45,17 @@ impl Ctx {
         let wasi = builder.build();
 
         // Create the wasi-nn context.
-        let mut openvino = backend::openvino::OpenvinoBackend::default();
+        #[cfg(feature = "openvino")]
+        let mut backend = backend::openvino::OpenvinoBackend::default();
+        #[cfg(feature = "winml")]
+        let mut backend = backend::winml::WinMLBackend::default();
```

Comment:
This won't work when both features are enabled. Features are additive, so both backends can be present at the same time.

Reply:
We run openvino and winml backend in two different test cases, with different inputs, so only one backend is specified here for testing purpose.

backend.list() is able to list all available backends.

https://github.com/bytecodealliance/wasmtime/blob/9732234291ffc9486b95be5309f920775b20d963/crates/wasi-nn/src/backend/mod.rs#L21-L31